src/js/components/award/shared/activity/ContractGrantActivityChartVerticalLines.jsx

```diff
@@ -101,7 +112,11 @@ const ContractGrantActivityChartVerticalLines = ({
                 showTextPosition="left"
                 adjustmentX={padding.left}
                 textClassname={`${endLineClassName}`}
-                lineClassname={`${endLineClassName}`} />}
+                lineClassname={lineData[2].classname}
+                onMouseMoveLine={showHideTooltip}
+                onMouseLeaveLine={showHideTooltip}
+                onMouseMoveText={showHideTooltip}
+                onMouseLeaveText={showHideTooltip} />}
```

Comment:
If you let this `SVGLine` component be the children of the tooltip, you shouldn't have to do this.